examples/grpc-client/main.go

```diff
@@ -64,17 +65,20 @@ func run() error {
 	if *parseDocument {
 		client := parserv1.NewParserServiceClient(conn)
 
+		filename := "examples/grpc-client/hello.md"
```

Comment:
Hint: it's ok, but it can be improved by using [`//go:embed`](https://pkg.go.dev/embed) so that it won't depend on from where `main.go` is run.